app.js

```diff
@@ -1,48 +1,73 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable class-methods-use-this */
```

Comment:
- [ ] Do not disable linters, remember they are there to help you. As I can see both of them can be fixed, so please update the files and remove the disabling

Reply:
Hello @GzimAsani, thank you for your feedback. We're trying to reach out to you in order to understand what you mean with "separate the classes", in our code we are using 2 classes ( `Book` and `Library` ) that are separeted, so we don't understand what do we have to separate? Do we have to place them in different files?
And also, about disabling some rules, we thougth it was allowed because in the documentation says:

> TIP: If you separated the code in multiple files and you you have the following issues with the linter: complaining about having multiple classes at the same file, or an unused variables, disable the linter check for that rule in the file with a /* eslint-disable rule-you-want-to-disable */ . 
> Check the [ESLint documentation](https://eslint.org/docs/latest/user-guide/configuring/rules#disabling-rules) for more details on how.

So that, we though it was fine to disable those specific rules.